pkgs/applications/misc/taskwarrior/default.nix

```diff
@@ -26,10 +26,11 @@ stdenv.mkDerivation rec {
   checkTarget = "test";
 
   postInstall = ''
-    mkdir -p "$out/share/bash-completion/completions"
-    ln -s "../../doc/task/scripts/bash/task.sh" "$out/share/bash-completion/completions/task.bash"
-    mkdir -p "$out/share/fish/vendor_completions.d"
-    ln -s "../../../share/doc/task/scripts/fish/task.fish" "$out/share/fish/vendor_completions.d/"
+    installShellCompletion --bash $out/share/doc/task/scripts/bash/task.sh
+    rm -r $out/share/doc/task/scripts/bash
+    installShellCompletion --fish $out/share/doc/task/scripts/fish/task.fish
+    rm -r $out/share/doc/task/scripts/fish
+    mv $out/share/doc/task/scripts/vim $out/share/vim-plugins
```

Comment:
So it comes with a vim plugin. Would be great to also wrap it under `vimPlugins`. `fzf` can be a reference:
https://github.com/NixOS/nixpkgs/blob/fbe74fa34622137b15904f6a3e3520f4f2daba6d/pkgs/applications/editors/vim/plugins/overrides.nix#L467-L476

Reply:
Thanks for the offer! Added `vimPlugins.taskwarrior`.

Reply:
we should be able to just remove this instead

Reply:
> we should be able to just remove the directory instead

Which directory? And why?

Reply:
`$out/share/doc/task/scripts/vim`, because I don't think moving it to `$out/share/vim-plugins` accomplishes anything since we already have a vim plugin

Reply:
> I don't think moving it to `$out/share/vim-plugins` accomplishes anything since we already have a vim plugin

Personally, and I want to believe there are more users like me out there, that don't use `vimPlugins` at all, and define all vim plugins imperatively. Hence I think all packages that provide non standard vim plugins like this, should put these vim files in a more predictable path.

We can argue about the path though, your comment made me take a nixpkgs survey with `nix-locate`. And:

- `share/vim-plugins`: 5 packages:
- - `txr`
- - `tamarin-prover`
- - `rosie`
- - `fzf`
- - `boogie`
- `share/vim/`: 9 packages:
- - `quick-lint-js`
- - `python39Packages.powerline`
- - `poke`
- - `libvarlink`
- - `firejail`
- - `arcanPackages.espeak`
- - `cmake`
- - `clisp`
- - `augeas`

Only `taskwarrior` installs these files to `share/doc/${pname}/scripts/vim`.

Considering also the `runtimepath`s of `vim` and `neovim`, I think it reached the conclusion that like `rosie`, I should also `ln -s $out/share/vim-plugins/task $out/share/nvim/site/task` which is pushed now.

Reply:
fair enough